SDK_versioned_docs/version-3.0.0/subgraph-example.md

```diff
@@ -0,0 +1,282 @@
+---
+id: Subgraph examples
+sidebar_position: 3
+---
+
+# Subgraph Query Examples
+You can use the subgraph to make GraphQL queries to fetch data about Uniswap V3. You can fetch data points like the spot price of a token, current liquidity of a pool, volume on a certain day, and much more. Below are some example queries. You can copy and paste them in the [v3 explorer](https://thegraph.com/hosted-service/subgraph/uniswap/uniswap-v3) to get fresh data.
+
+## Global Data
+To query global data, pass in the Uniswap Factory address : `0x1F98431c8aD98523631AE4a59f267346ea31F984` and select the desired fields.  Reference the full [factory schema](https://github.com/Uniswap/v3-subgraph/blob/main/schema.graphql#L1) to see all possible fields.
+
+### Current Global Data
+An example querying total pool count, transaction count, and total volume in USD and ETH:
```

Comment:
sorry I'm a noob - when we say "an example querying" do we mean the query will output those data points?